README.md

```diff
@@ -165,6 +166,12 @@ For example, a second release in June 2021 would be 202106.01. Although the SDK
 
 All of the released versions of the C-SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.1.2](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.2).
 
+### 202012.00
+
+[API documentation of 202012.00 release](https://docs.aws.amazon.com/embedded-csdk/202012.00/lib-ref/index.html)
+
+This release includes AWS IoT Over-the-air Update, backoffAlgorithm, and corePKCS11 libraries. Additionally, there is a major update to the coreJSON and coreHTTP APIs. All libraries continue to undergo code quality checks (e.g. MISRA-C compliance), Coverity static analysis, and validation of memory safety with the C Bounded Model Checker (CBMC) automated reasoning tool.
```

Comment:
Yes. We can say like this "This release includes new libraries which includes AWS IoT Over-the-air Update (Release Candidate), backOffAlgorithm and corePKCS11.".